### DISCUSSION TITLE
@RaywangQvQ 要不要考虑搭建一个文档页

### DISCUSSION THREAD

**Discussion by sysfox** [RayWangQvQ/BiliBiliToolPro#117] [Ideas]
要不要讨论一下？

**Comment by RayWangQvQ** [comment]
是指wiki吗？还是以github为域名的站点？

**Reply by sysfox** [reply]
以github为域名(甚至可以考虑vercel&netlify)，解决部分用户无法加载图片的问题。
ps：可以考虑下使用这两个免费域名项目中的一个[js.org](https://github.com/js-org/js.org) 或[js.cool](https://github.com/willin/js.cool) ,方便用户访问以及cdn加速

**Reply by RayWangQvQ** [reply]
单独域名的文档站点太高调了点，如果只是为了解决国内局域网的访问问题，倒是可以考虑把仓库同步一份到gitee

**Reply by sysfox** [reply]
同意,不过个人觉得同步到gitee感觉还要张扬(doge)

**Reply by RayWangQvQ** [reply]
哈哈哈好像确实是这样，而且现在gitee已经开始对public的仓库进行审核，审核通过才能public😂
